### ISSUE TITLE
Error after plugin update

### ISSUE THREAD

**Issue by Melknix** [microsoft/azure-tools-for-java#5885]
IntelliJ build version: 2021.2.2 IU-212.5284.40
OS: Windows 10
JDK: JetBrains s.r.o. 11.0.12
Plugin version: 3.57.0-2021.2
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,8,1921x1048,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.Throwable: Conflicting component name 'RunManager': class com.intellij.execution.impl.RunManagerImpl and class com.intellij.execution.impl.RunManagerImpl (componentManager=Project(name=tellus-gms-2.0-springboot, containerState=COMPONENT_CREATED, componentStore=C:\progetti\tellus-gms-2.0-springboot))
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:182)
	at com.microsoft.azure.toolkit.intellij.connector.ConnectionRunnerForRunConfiguration$MyRunConfigurationExtension.isApplicableFor(ConnectionRunnerForRunConfiguration.java:103)
```

**Comment by wangmingliang-ms** [comment]
1. does this error cause any failures? 
2. did you create any Azure Resource Connections(consume Azure Resources from your project via Azure Resource Connector feature) in this project?

**Comment by Melknix** [comment]
1. Yes, after plugin update every time I open a project have this error, this error cause the failure of load the running configuration of the project (top-right). I had disabled the pugin and now I can see all the running configuration.
2. No. My project not using ARC.

**Comment by wangmingliang-ms** [comment]
@Melknix this bug is fixed in the latest `3.57.1`
